src/foam/nanos/auth/User.js

```diff
@@ -347,3 +347,25 @@ foam.RELATIONSHIP({
     tableWidth: 120
   }
 });
+
+// Relationship used in the agent auth service. Determines permission list when acting as a super user.
+foam.RELATIONSHIP({
+  cardinality: '*:*',
+  sourceModel: 'foam.nanos.auth.User',
+  targetModel: 'foam.nanos.auth.User',
+  forwardName: 'superUsers',
```

Comment:
Would 'agentcies' be a better word since the forward and inverse names should be the inverse of each other.